playbooks/roles/shadowsocks/tasks/main.yml

```diff
@@ -17,27 +11,23 @@
   command: cat {{ shadowsocks_password_file }}
   register: shadowsocks_password
 
-- name: Install Shadowsocks
-  pip: name=shadowsocks
+- name: Add the Shadowsocks-libev APT key
+  apt_key: url=http://shadowsocks.org/debian/1D27208A.gpg state=present
+  
+- name: Add the Shadowsocks-libev repository
+  apt_repository: repo='deb http://shadowsocks.org/ubuntu trusty main' state=present
```

Comment:
We would want to use the gathered Ansible facts in place of `ubuntu` and `trusty` in this line. [Here](https://github.com/jlund/streisand/blob/master/playbooks/roles/nginx/tasks/main.yml#L18) is an example of how that is being done in the Nginx role.

For the sake of future upgrades (e.g. to Ubuntu 16.04) we want to avoid having as many static references to the current LTS Trusty codename as we possibly can.